torchquad/integration/vegas_map.py

```diff
@@ -180,35 +180,30 @@ def update_map(
         """Update the adaptive map, Section II C."""
         self._smooth_map()
 
-        # Initialize new locations
```

Comment:
Does it affect gradients?

Gradient tests fails for me for VEGAS():

grad=tensor([[-1.7841,  2.2152]])

gradient_tolerance = 5e-2

Reply:
The reason is likely that some of the parameters in VEGAS changed like N_strat. So for this particular test errors are a bit higher now. I increased N that reduces the error below the defined bounds, so I think it should be fine? (converges properly it seems)

Reply:
For additional validation I am rerunning python vegas vs TQ vegas comparisons. Previous version 
![image](https://user-images.githubusercontent.com/2656685/128490042-6ac89267-9f33-41fe-9a57-d7929a7c84d3.png)

Reply:
New version 
![image](https://user-images.githubusercontent.com/2656685/128490111-c2aa1478-0a21-4dfa-b966-1f835fc7c12f.png)

Reply:
So I think we are fine?

Reply:
Yes, I don't have errors anymore.